core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -544,15 +544,12 @@ public Mono<GuildEmoji> getGuildEmojiById(final Snowflake id, EntityRetrievalStr
     }
 
     /**
-     * Gets the enabled guild features.
-     * <br>
-     * You can see the available
-     * <a href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features">guild features</a>
+     * Returns the enabled features of this {@link Guild}.
```

Comment:
Maybe can mention here you can still get the "raw" data features using `#getData.features()`